lib/Criteria/VideoQueryCriteria.php

```diff
@@ -17,6 +17,12 @@ class VideoQueryCriteria
     const SORT_ORDER_DESC = 0;
     const SORT_ORDER_ASC = 1;
 
+    const PUBLICATION_STATES = [
+        'published' => 1,
```

Comment:
It would be better to create some other constants for these static strings. Then you can have auto completion for possible values